lte/gateway/python/scripts/generate_oai_config.py

```diff
@@ -282,6 +285,13 @@ def _get_context():
         "spgw", "s1u_iface_name"
     )
 
+    try:
+        sgw_s5s8_up_ip = get_ip_from_if_cidr(iface_name, IpPreference.IPV4_ONLY)
+    except ValueError:
+        # ignore the error to avoid MME crash
```

Comment:
I would suggest adding a warning log to specify the IP couldn't be parsed and thus it was overridden